app/assets/stylesheets/ama_layout/layout_components/banner.scss

```diff
@@ -0,0 +1,29 @@
+.banner{
+	position: relative; /* This is needed for the positioning of the close icon on the blue banners*/
+
+	&--blue{
+		background: $brand-blue-insurance;
+		padding: 20px;
+	}
+
+	&--red{
+		background: $brand-red;
+    padding: 42px 20px;
+    color: $white;
+	}
+
+  &__link{
+    text-align: center;
+    display: block;
+  
+    a{
```

Comment:
we want to avoid using tag selectors (it increases specificity unnecessarily, and is costlier in terms of page speed). Also .banner__link should reference the a tag itself, so:
<div class="banner">
<a class="banner__link">waffles</a>
</div>